package.json

```diff
@@ -7,10 +7,10 @@
   "author": "Marlon Keating <marlonakeating@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "webpack --config webpack.config.js && npm run build:sass && python manage.py collectstatic --noinput",
+    "build": "webpack --config webpack.config.js && npm run build:sass && python manage.py collectstatic --noinput -i *.scss",
```

Comment:
As part of trying to figure out what I want webpack to do and what I want django to do in terms of deployment, I came across [this SO post](https://stackoverflow.com/questions/8269883/how-to-ignore-directories-when-running-django-collectstatic) - `collectstatic` should no longer bring over .scss partials to the production static directory. We can extend that with multiple -i flags if there are other things it brings over that we don't want.

console output after an npm run build should read similar to this:
```
Copying '/home/dev-admin/git/CivicTechExchange/civictechprojects/static/css/compiled/styles.css'
Copying '/home/dev-admin/git/CivicTechExchange/common/static/js/bundle.js'
Copying '/home/dev-admin/git/CivicTechExchange/common/static/js/bundle.js.map'

3 static files copied to '/home/dev-admin/git/CivicTechExchange/staticfiles', 109 unmodified.
```

rather than copying 100+ as it brings over all the scss partials.